lib/datasource/crate/index.ts

```diff
@@ -10,6 +10,7 @@ import { privateCacheDir, readFile } from '../../util/fs';
 import { Http } from '../../util/http';
 import * as cargoVersioning from '../../versioning/cargo';
 import type { GetReleasesConfig, Release, ReleaseResult } from '../types';
+import { CrateRecord, RegistryFlavor, RegistryInfo } from './types';
```

Comment:
```suggestion
import type { CrateRecord, RegistryFlavor, RegistryInfo } from './types';
```

?

Reply:
Doesn't work, as `RegistryFlavor` is an enum 😕